run-tests.sh

```diff
@@ -2,43 +2,49 @@
 
 set -e
 
-DOCKERFILE_TEMPLATE="tests/Dockerfile.template"
-
 PY_DOCKER_IMAGES=()
-PY_DOCKER_IMAGES+=("2.7.16-slim")
-PY_DOCKER_IMAGES+=("3.7.4-slim")
-PY_DOCKER_IMAGES+=("3.8-slim")
-PY_DOCKER_IMAGES+=("3.9-slim")
-PY_DOCKER_IMAGES+=("3.10-slim")
-PY_DOCKER_IMAGES+=("3.11-slim")
-
-create_docker_file() {
-    local package="$1"
+if [ -n "$TEST_PY_VERSION" ]; then
+	PY_VERSIONS+=("$TEST_PY_VERSION")
+else
+	PY_VERSIONS+=("2.7")
+	PY_VERSIONS+=("3.7")
+	PY_VERSIONS+=("3.8")
```

Comment:
These two are EOL. Do you still support them?

Reply:
I would leave that decision up to @mmicu - I'm fine with keeping EOLed versions as long as there is no technical reason breaking them.
After all editorconfig-checkers place is of a supporting role, maybe helping people slowly transition to newer python versions, and when the only weight for us is some minutes spent running the actions, I do not consider this a cost worth saving. Of cause this argument goes out of the window as soon as we are blocked by a problem that only occurs on EOL versions.